PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,12 @@
 # Description
 
-(Delete this, but please include a summary of the change and which issue is fixed -include its number-. Please also include relevant motivation and context.)
+*(Delete this, but please include a summary of the change and which issue is fixed -include its number-. Please also include relevant motivation and context.)*
 
 # Checklist:
 
 - [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
+  - [ ] This PR only contains configuration changes (package.json, etc.)
```

Comment:
nit: White space at beginning of line ;) Please align with the rest!

Reply:
Hooooo, now I see that these are probably intentional? Do they actually indent?

Reply:
it indents, just confirmed :)